contracts/UniswapOracle.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity ^0.6.6;
+pragma experimental ABIEncoderV2;
+
+import "./UniswapContracts.sol";
+
+/**
+ * @notice Uniswap Oracle based on https://github.com/Uniswap/uniswap-v2-periphery/blob/master/contracts/examples/ExampleOracleSimple.sol.
+ * @dev A simple oracle for storing TWAP for 6 assets pairs - ETH, WBTC, ZRX, BAT, REP, DAI.
+ * @author Compound Labs, Inc.
+ */
+contract UniswapOracle {
```

Comment:
should we make a `OpenOraclePriceData` (or something) interface, and have the uniswap oracles and `anchorOracle` inherit from it? kind of hard to follow the interfaces here, they dont appear to match in some places

Reply:
Yes, it should be done. It was not clear how it will be used as an anchor or source. I can definitely adjust now.